e2e/features/CreateUserRole.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Join Group
+  In order to join a group
+  As a user
+  I want to search a group and join it
+  Background:
+    Given I'm in the homepage
+    And I log in as "default1" with password "password"
+    Then The "Register" menu is not present
+
+  Scenario: Join a group
+    Given I am in group-list page "1"
+    Then I click the "Join" button
```

Comment:
This would be a When step, after which a Then step should check the intended outcome, for instance that the user is now among the group members